pkg/integration/utils.go

```diff
@@ -27,6 +27,16 @@ func GetExitCode(err error) (int, error) {
 			return procExit.ExitStatus(), nil
 		}
 	}
+	// error if contained command not found/does not exist
```

Comment:
not ignoring this, just looking into it more to compare to derr.Errs...not very easy to test this package, actually.